fastlane/Fastfile

```diff
@@ -70,15 +70,15 @@ ENV["validate_translations"]="lintVanillaRelease"
 
     android_bump_version_release()
     new_version = android_get_app_version()
-    extract_release_notes_for_version(version: new_version, 
-      release_notes_file_path:"#{ENV["PROJECT_ROOT_FOLDER"]}RELEASE-NOTES.txt", 
+    extract_release_notes_for_version(version: new_version,
+      release_notes_file_path:"#{ENV["PROJECT_ROOT_FOLDER"]}RELEASE-NOTES.txt",
```

Comment:
Nitpick. I'm pretty sure this was a change from your editor, but since it happens to touch `PROJECT_ROOT_FOLDER`, maybe we can adopt Ruby `File.join` syntax here, too, like later on in the file?

```suggestion
      release_notes_file_path: File.join(ENV["PROJECT_ROOT_FOLDER"], 'RELEASE-NOTES.txt'),
```